public/data/emails.json

```diff
@@ -0,0 +1,20 @@
+
+{
+  "emails":[
+    {
+      "emailId": 1,
+      "receivedTime": "August 11, 2020 23:15:30",
+      "lastRead": null,
```

Comment:
This might need a little rework since there are no sub messages with dates to associate with